modules/administration/pages/tshoot-registerclones.adoc

```diff
@@ -19,17 +19,25 @@ If more detailed instructions are required, put them in a "Resolving" procedure:
 . Last step
 ////
 
-Sometimes a cloned client (either traditional or Salt) will use the same machine ID as the system they are a clone of.
-This results in {productname} only recognizing one system, rather than two different systems.
-This can be resolved by changing the machine ID of the  cloned system, so that {productname} recognizes them as two different clients.
+
+If you are using {productname} to manage virtual machines, you might find it useful to create clones of your VMs.
+A clone is a VM that uses a primary data volume that is an exact copy of an existing data volume.
```

Comment:
I woudn't call this a data volume (ok, everything is data...).  You primiliary clone the volume with the system data (the OS).
If you agree we must change the surrounding text, too.